postgresql/Dockerfile

```diff
@@ -30,5 +30,6 @@ USER root
 RUN addgroup postgres --gid 1001 && useradd -s /sbin/nologin --gid 1001 --uid 1001 postgres
 
 RUN chown -R 1001:1001 /opt/bitnami
+RUN chown -R 1001:1001 /bitnami/postgresql
```

Comment:
@pgvishnuram what do you think about this change? Seems like it has the potential to break existing installations.

Reply:
Removed this permission. It was added earlier to provision permissions that allows to run docker image for debugging and testing. I worked to run them other way so no need for this permission further.